Keep in mind, that Kiwi (2020-10-08) changed its overlay path into:
         source: "/run/overlay/rootfsbase"
With older Kiwi, please use:
         source: "/run/rootfsbase"
See also:
https://github.com/OSInside/kiwi/issues/1558
https://github.com/OSInside/kiwi/commit/89f85e45041eb39f686f6b20f909a20bb48277cc#diff-473d8885cf8c5da1798144d6eaeb94c386af31c776c34336331040ca49b667d8

diff -ruN a/src/modules/unpackfs/unpackfs.conf b/src/modules/unpackfs/unpackfs.conf
--- a/src/modules/unpackfs/unpackfs.conf
+++ b/src/modules/unpackfs/unpackfs.conf
@@ -88,13 +88,15 @@
 # for its destination name, as in the example below.
 
 unpack:
-    -   source: ../CHANGES
-        sourcefs: file
-        destination: "/tmp/changes.txt"
-        weight: 1  # Single file
-    -   source: src/qml/calamares/slideshow
-        sourcefs: file
-        destination: "/tmp/slideshow/"
-        exclude: [ "*.qmlc", "qmldir" ]
-        weight: 5  # Lots of files
-        # excludeFile: /etc/calamares/modules/unpackfs/exclude-list.txt
+#   -   source: ../CHANGES
+#       sourcefs: file
+#       destination: "/tmp/changes.txt"
+    -   source: "/run/overlay/rootfsbase"
+        sourcefs: "ext4"
+        destination: "/"
+        exclude: [ "/etc/dracut.conf.d/02-livecd.conf" ]
+#   -   source: src/qml/calamares/slideshow
+#       sourcefs: file
+#       destination: "/tmp/slideshow/"
+#       exclude: [ "*.qmlc", "qmldir" ]
+#       excludeFile: /etc/calamares/modules/unpackfs/exclude-list.txt
